WLC/utils/azure.py

```diff
@@ -0,0 +1,76 @@
+import hashlib
+import logging
+import os
+
+import cv2
+from azure.storage.blob import BlockBlobService, ContentSettings
+
+LOGGER = logging.getLogger()
+
+# Azure spams the logs, this will make it quiet.
+logging.getLogger("azure").setLevel(logging.CRITICAL)
+
+
+def get_block_blob_service():
+    if 'BLOB_ACCOUNT' not in os.environ or 'BLOB_KEY' not in os.environ:
```

Comment:
You can do something similar to what @PaulSonOfLars suggested here as well.

Reply:
I'd argue this is ok, as you're just checking the presence - the other situation was checking its presence AND evaluating its value.